Integrations/integration-Cloaken.yml

```diff
@@ -0,0 +1,79 @@
+description: 'Unshorten urls through a tor proxy server. Unshorten urls onsite with
+  the power of Tor to prevent leaking ip addresses to adversaries.  Created by CypherInt.  Visit
+  us at https://cypherint.com.  Subscribe on the Amazon marketplace: https://aws.amazon.com/marketplace/'
+detaileddescription: cloaken will return the unshortened version of a url
+configuration:
```

Comment:
Please add `proxy` parameter

Reply:
I will have to refactor the SDK and rebuild the docker.  I will work on this next.  Thanks for catching this.

Reply:
I added proxy parameter